src/products/filters/models/ProductsFilter.ts

```diff
@@ -0,0 +1,16 @@
+import AlphabeticCurrencyCode from '../../../model/AlphabeticCurrencyCode';
+import ChannelApeApiError from '../../../model/ChannelApeApiError';
+
+export default interface ProductsFilter {
```

Comment:
I'm confused why this is needed? It looks like a duplicate of the product filter

Reply:
I think this happened when I renamed something, I'll remove.